engines/bastion_katello/app/assets/javascripts/bastion_katello/packages/details/views/package-info.html

```diff
@@ -9,22 +9,31 @@ <h4 translate>Package Information</h4>
       <dd>
         <i class="fa fa-spinner fa-spin" ng-show="installedPackageCount === undefined"></i>
         <span ng-show="installedPackageCount !== undefined">
-          <a href="{{ encodedUrl('installed_package') }}" translate>
+          <a ng-if="newHostDetailsUI" href="{{ newHostUrl('installed_package') }}" translate>
+            {{ installedPackageCount }} Host(s)
+          </a>
+          <a ng-if="!newHostDetailsUI" href="{{ encodedUrl('installed_package') }}" translate>
```

Comment:
For some reason, even if I set the new host details page to not be used in the settings, I always see the link to the  hosts index page rather than the content hosts page.

This works however for the bulk errata...

Reply:
Perhaps `newHostDetailsUI` is always true for whatever reason?

Reply:
It's not always `true` but it's always truthy! :smile: